### ISSUE TITLE
panic: reached unreachable code (bun pm trust)

### ISSUE THREAD

**Issue by baphony** [oven-sh/bun#10298]
### What version of Bun is running?

1.1.3+2615dc742

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

![image](https://github.com/oven-sh/bun/assets/112877883/5d75a958-c5dc-4147-8922-201d78434ffa)
```shell
bun pm trust --all
``` 

Or

```shell
bun pm trust jss
``` 

```
Bun v1.1.3 (2615dc74) Windows x64
PackageManagerCommand: bunfig spawn lifecycle_scripts
Elapsed: 236ms | User: 15ms | Sys: 31ms
RSS: 59.33MB | Peak: 59.33MB | Commit: 99.23MB | Faults: 14661
----- bun meta -----
0   00007FF6A7473E18
1   ???
2   ???
3   ???
4   ???
5   ???
6   ???
7   ???
8   ???
thread 7744 panic: reached unreachable code
???:?:?: 0x7ff6a605347a in ??? (bun.exe)
???:?:?: 0x7ff6a6056ba9 in ??? (bun.exe)
???:?:?: 0x7ff6a60a45a0 in ??? (bun.exe)
???:?:?: 0x7ff6a5dd1603 in ??? (bun.exe)
???:?:?: 0x7ff6a8973eb3 in ??? (bun.exe)
???:?:?: 0x7fffe8e8257c in ??? (KERNEL32.DLL)
???:?:?: 0x7fffeb08aa47 in ??? (ntdll.dll)
``` 

```shell
bun pm trust postinstall-postinstall
``` 

``` 
Elapsed: 54ms | User: 15ms | Sys: 31ms
RSS: 52.84MB | Peak: 52.84MB | Commit: 92.84MB | Faults: 13073
----- bun meta -----
0   00007FF6A7473E18
1   ???
2   ???
3   ???
4   ???
5   ???
6   ???
7   ???
8   ???
thread 17932 panic: reached unreachable code
???:?:?: 0x7ff6a605347a in ??? (bun.exe)
???:?:?: 0x7ff6a6056ba9 in ??? (bun.exe)
???:?:?: 0x7ff6a60a45a0 in ??? (bun.exe)
???:?:?: 0x7ff6a5dd1603 in ??? (bun.exe)
???:?:?: 0x7ff6a8973eb3 in ??? (bun.exe)
???:?:?: 0x7fffe8e8257c in ??? (KERNEL32.DLL)
???:?:?: 0x7fffeb08aa47 in ??? (ntdll.dll)
``` 

```shell
bun pm trust core-js-pure
``` 

``` Elapsed: 72ms | User: 0ms | Sys: 62ms
RSS: 52.87MB | Peak: 52.87MB | Commit: 92.85MB | Faults: 13081
----- bun meta -----
0   00007FF6A7473E18
1   ???
2   ???
3   ???
4   ???
5   ???
6   ???
7   ???
8   ???
thread 19448 panic: reached unreachable code
???:?:?: 0x7ff6a605347a in ??? (bun.exe)
???:?:?: 0x7ff6a6056ba9 in ??? (bun.exe)
???:?:?: 0x7ff6a60a45a0 in ??? (bun.exe)
???:?:?: 0x7ff6a5dd1603 in ??? (bun.exe)
???:?:?: 0x7ff6a8973eb3 in ??? (bun.exe)
???:?:?: 0x7fffe8e8257c in ??? (KERNEL32.DLL)
???:?:?: 0x7fffeb08aa47 in ??? (ntdll.dll)
``` 

Same thing for all the postinstall and install scripts.

**Comment by dylan-conway** [comment]
Can you share your package.json? I'm curious if this still happens if you delete node_modules and reinstall

**Comment by baptistecs** [comment]
I did try to remove all node_modules folders but I get the same result :-/

Unfortunately I can’t share our package.json for security reasons, there is the list of the packages and their version in the screenshot if that can help?

**Comment by baphony** [comment]
I have the same issue at home:
![image](https://github.com/oven-sh/bun/assets/112877883/79f5e3a9-b16e-4a66-a5d8-9737f7d449ad)
![image](https://github.com/oven-sh/bun/assets/112877883/aa6047e5-ac23-4046-88bd-242879f51a0b)
Microsoft Windows NT 10.0.22631.0 x64
1.1.3+2615dc742

**Comment by dylan-conway** [comment]
Fixed with #12215 in bun v1.1.18

**Comment by baphony** [comment]
Many thanks! :)
